src/components/templates/News.tsx

```diff
@@ -9,9 +9,9 @@ const News: React.FC = observer(() => {
     const store :RootStore = useStore();
     return (<Box flexDirection={'column'}>
         {store.newsFlashCollection.map((news) => {
-            const date :Date = new Date(news.date);
+            const date :null|string|Date = news.date ==null ? '': new Date(news.date).toLocaleDateString();
```

Comment:
`Date` type is not required, according to your code, `date` can be either `null` or `string`

Reply:
done, I did it because mock data was in type `Date`, so I change mock data to type `string`